index.md

```diff
@@ -88,13 +88,18 @@ Make your event and your presentations accessible so that you are prepared for s
 
 Presentations that are accessible to people with disabilities are also **inclusive** to many more audiences. For example, people with different learning styles and people who are not fluent in the language.
 
-Accessible presentations also have additional benefits, such as in these situations:
+Accessible presentations also have additional benefits, such as in this situation:
```

Comment:
I think this is still a 'there situations' as there are a couple of points mentioned

Reply:
I was trying to make it a single story -- it's one presentation / one situation, and they do all of those things. :-)

Unbulleting it will help that make more sense.